### ISSUE TITLE
NTR: MR or CT: ring-enhancing lesion of the central nervous system (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8894]
New term request MR or CT: ring-enhancing lesion of the central nervous system (jadhav_091112184714):   UMLS: 0742470

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000919
- label: Ring-enhancing cerebral lesion
- parents: [Abnormal cerebral morphology (HP:0002060)](https://hpo.jax.org/app/browse/term/HP:0002060)
- pmid: PMID:28198691
- def: An abnormal neuroimaging finding consisting in an area of decreased density surrounded by a bright rim from concentration of the enhancing contrast dye.
- comment: See Figure 1 of PMID:28198691.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8894
- synonyms:
